lib/client.js

```diff
@@ -84,6 +84,35 @@ try {
   channels.connected = { hasSubscribers: false }
 }
 
+let fastNow = Date.now()
+const fastTimers = []
+const fastNowInterval = setInterval(() => {
+  fastNow = Date.now()
+
+  let len = fastTimers.length
+  let idx = 0
+  while (idx < len) {
+    const timer = fastTimers[idx]
+    if (timer.timeoutType === TIMEOUT_DESTROY) {
+      if (idx !== len - 1) {
+        fastTimers[idx] = fastTimers.pop()
+      } else {
+        fastTimers.pop()
+      }
+      len = fastTimers.length
+    } else {
+      if (timer.timeoutExpires && fastNow >= timer.timeoutExpires) {
+        timer.timeoutExpires = 0
+        onParserTimeout(timer)
+      }
+      idx += 1
+    }
+  }
+}, 1e3)
```

Comment:
This should be active only if there is an http going on. Given how this is implemented, every node process would get this timer.